src/main/rules/GCI1024/IaC/GCI1024.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
the path of this file is wrong ...
please check another example 
JSON file will be inside root directory of "GCI1024"